conda_envs/ooi_windows.yml

```diff
@@ -255,4 +255,4 @@ dependencies:
     - tqdm==4.45.0
     - widgetsnbextension==3.5.1
     - xarray==0.15.1
-prefix: C:\Users\fschwock\AppData\Local\Continuum\anaconda3\envs\obspy
\ No newline at end of file
+prefix: C:\Users\fschwock\AppData\Local\Continuum\anaconda3\envs\obspy
```

Comment:
looks like this is pointing to something on my local machine. Do we need this line or does anybody know how we can fix that?

Reply:
with all of the correct conda / pip installation working, do we need these environment files anymore?

Reply:
that's true. We can probably remove those yml files.

Reply:
I can remove them. I found another thing that looked a bit awkward after going through the linting.

Reply:
Just realized it might be easier to merge this PR first and then do those chnages

Reply:
sounds good to me.